test/integration/targets/vmware_content_library/aliases

```diff
@@ -0,0 +1,2 @@
+cloud/vcenter
+shippable/vcenter/group1
```

Comment:
Please add:

```needs/target/prepare_vmware_tests```

this way, the `prepare_vmware_tests` role will be available during the test.

Reply:
Sure I will make this change